### DISCUSSION TITLE
How do I create a user list and how do I download using it?

### DISCUSSION THREAD

**Discussion by c0mplexx92** [Serene-Arc/bulk-downloader-for-reddit#556] [Q&A] [answered]
So I noticed it supports user lists but i'm not sure how to go on about making the list and to download using it

**Comment by Serene-Arc** [comment, marked as answer]
What are you looking for? You can't directly supply a text file of users but the `--user` option can be supplied multiple times.

**Reply by c0mplexx92** [reply]
Yeah I was thinking of consolidating a few usernames into 1 command/word but I guess I would've needed to supply a text file for that

**Comment by Hoover889** [comment]
if you are running this on Windows you can set up a simple batch file to download a list of user's posts (just change the directory to wherever you want to save the files.)

`FOR /F "usebackq tokens=*" %%A in (M:\Archive\UserList.txt) do py -m bdfr download M:\Archive\%%A --user %%A --submitted --folder-scheme "" --file-scheme {SUBREDDIT}_{TITLE}_{POSTID} --no-dupes --search-existing`
